src/xcode/ENA/ENA/Source/Workers/Logging/Logging.swift

```diff
@@ -29,8 +29,10 @@ extension OSLog {
 	static let survey = OSLog(subsystem: subsystem, category: "survey")
 	/// PP Analytics
 	static let ppa = OSLog(subsystem: subsystem, category: "ppa")
-	/// Events
-	static let checkin = OSLog(subsystem: subsystem, category: "checkin")
+	/// Event registration - checkin / attendees
```

Comment:
Cant we just call it "Event". Is the "registration" needed?

Reply:
So then we have "EventCheckin" instead of "evregCheckin"

Reply:
Event as name is so potentially misleading, because it also could be an system event or something similar.
So I think Nicks suggestion avoids this problem in a smart way :)